src/plugins/blocksmith/components/eme-form-block-modal/index.jsx

```diff
@@ -37,11 +39,16 @@ export default class EmeFormBlockModal extends React.Component {
       touched: false,
       isValid: block.has('form_ref'),
       formRef: block.get('form_ref'),
+      hasExpireDate: block.has('expires_at'),
+      expireDate: block.get('expires_at', new Date()),
```

Comment:
@albertgcsula why not call this `expiresAt`?

Reply:
done